src/azul/lambda_layer.py

```diff
@@ -178,6 +189,23 @@ def _parse_reqs(self, reqs: Path) -> Iterable[Requirement]:
 
     @cached_property
     def object_key(self):
-        sha = '.'.join(file_sha1(f)
-                       for f in (self.reqs_file, self.reqs_trans_file))
-        return f'{config.lambda_layer_key}/{sha}.zip'
+        # We include requirements.txt, requirements.trans.txt, and vendored
+        # wheels in the hash to keep the layer content-addressable. The other
+        # files don't reference content in the layer, but we include them
+        # regardless since they may affect how the layer is generated. For
+        # example, Chalice is a dev-requirement, but updating may affect how
+        # dependencies are packaged.
+        relevant_files = (
+            self.reqs_file,
+            self.reqs_trans_file,
+            *self.wheel_dir.iterdir(),
+            self.reqs_pip_file,
+            self.reqs_dev_file,
+            __file__,
+            self.root / 'scripts' / 'stage_layer.py'
+        )
+        sha1 = hashlib.sha1()
+        for file in relevant_files:
+            sha1.update(file_sha1(file).encode())
+        hash_of_hashes = sha1.hexdigest()
```

Comment:
Don't see how this is a hash of hashes. It's just a hash of all the files' content. I don't really see the need for a variable, the expression is only used once and only two chars longer than the variable name.

Reply:
`file_sha1()` returns the hash of a file. So while the variable name is accurate, in-lining it is no problem.

Reply:
Makes sense.